requests/OmgEthPrice.js

```diff
@@ -0,0 +1,49 @@
+import * as Witnet from "witnet-requests"
+
+// Retrieves ETH price of OMG from the Binance API
+const binance = new Witnet.Source("https://api.binance.com/api/v3/ticker/price?symbol=OMGETH")
+  .parseJSONMap() // Parse a `Map` from the retrieved `String`
+  .getFloat("price") // Get the `Float` value associated to the `price` key
+  .multiply(1000000) // Use 6 digit precision
+  .round() // Cast to integer  
+
+// Retrieves ETH price of OMG from the Gate.io API
+const gateio = new Witnet.Source("https://data.gateapi.io/api2/1/ticker/omg_eth")
+  .parseJSONMap() // Parse a `Map` from the retrieved `String`
+  .getFloat("last") // Get the `Float` value associated to the `last` key
+  .multiply(1000000) // Use 6 digit precision
+  .round() // Cast to integer
```

Comment:
We should add more sources to this feed. Kraken, KuCoin, and Bittrex also have OMG/ETH.